web/python/make_web.py

```diff
@@ -21,18 +21,21 @@ def __missing__(self, key):
              'distributions': 'Distributions',}.items():
     FOLDER_NAMES[k] = v
 
-metadata = {}
-if os.path.exists('metadata.yaml'):
-    with open('metadata.yaml') as f:
-        metadata = yaml.load(f, Loader=yaml.FullLoader)
-if 'key4hep-spack' in metadata:
-    metadata['key4hep-spack'] = [metadata['key4hep-spack'], f"https://github.com/key4hep/key4hep-spack/commit/{metadata['key4hep-spack']}"]
-if 'spack' in metadata:
-    metadata['spack'] = [metadata['spack'], f"https://github.com/spack/spack/commit/{metadata['spack']}"]
-for k, v in metadata.items():
-    if isinstance(v, str) or len(v) == 1:
-        metadata[k] = [v, None]
-print(metadata)
+def get_metadata(folder_path):
```

Comment:
Now that I remember it, for metadata it would be fine to remove the `key4hep-spack` and `spack` commits; it's not necessary for users to know it and it's already bundled in the stack, so knowing which stack was used lets you know which commits were used. When I wrote this initially, the information of the commits wasn't available in each stack.